src/os/bluestore/Allocator.cc

```diff
@@ -14,7 +15,7 @@ Allocator *Allocator::create(CephContext* cct, string type,
   if (type == "stupid") {
     return new StupidAllocator(cct);
   } else if (type == "bitmap") {
-    return new BitMapAllocator(cct, size, block_size);
```

Comment:
I think we should just remove the old implementation since it's not even accessible.  That also lets you rename the class BitmapAllocator